plugin/PlayAssetDelivery/src/main/java/com/google/play/core/godot/assetpacks/PlayAssetDelivery.java

```diff
@@ -162,6 +193,34 @@ public Dictionary getAssetLocation(String packName, String assetPath) {
     return PlayAssetDeliveryUtils.convertAssetLocationToDictionary(retrievedAssetLocation);
   }
 
+  /**
+   * Calls fetch(List<String> packNames) method in the Play Core Library. Requests to download the
+   * specified asset packs. Emits fetchSuccess and fetchError signals when the underlying task
+   * succeeds/fails.
+   *
+   * @param packNamesArray String Array for all the packs to be fetched
+   * @param signalID identifier used to track mapping of signals to Tasks
+   */
+  public void fetch(final String[] packNamesArray, int signalID) {
```

Comment:
nit: does this need to be marked `final`?

Reply:
I double checked and they can be removed. (I initially thought this was required for passing in Arrays)